openmc/source.py

```diff
@@ -292,6 +294,8 @@ class IndependentSource(SourceBase):
         Energy distribution of source sites
     time : openmc.stats.Univariate or None
         time distribution of source sites
+    weight : float or None
+        weight value of source sites
```

Comment:
Is this still used?

Reply:
No, thank you for checking. It's already deleted.